Example/Player Example/ViewController.swift

```diff
@@ -78,17 +78,17 @@ extension ViewController: DMPlayerViewControllerDelegate {
   fileprivate func toggleFullScreen() {
     isPlayerFullscreen = !isPlayerFullscreen
     updateDeviceOrientation()
-    updatePlayerSize()
+    updatePlayerSize(self.view.frame.size)
```

Comment:
and then here pass in just the height. No explicit `self.` reference please: `updatePlayer(height: view.frame.size.height)`